ddtrace/tracer/option.go

```diff
@@ -138,19 +138,25 @@ func newConfig(opts ...StartOption) *config {
 		c.version = ver
 	}
 	if v := os.Getenv("DD_TAGS"); v != "" {
-		for _, tag := range strings.Split(v, ",") {
-			tag = strings.TrimSpace(tag)
+		sep, trim := " ", ","
+		if strings.Index(v, ",") > -1 {
+			sep, trim = trim, sep
+		}
+		for _, tag := range strings.Split(v, sep) {
+			tag = strings.Trim(tag, trim)
```

Comment:
```suggestion
			tag = strings.TrimSpace(tag)
```
Should be good enough in all cases, even if it's redundant when we choose space as a separator.